vancouver-drsm.csl

```diff
@@ -0,0 +1,466 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style class="in-text" version="1.0" xmlns="http://purl.org/net/xbiblio/csl" demote-non-dropping-particle="sort-only" page-range-format="minimal" default-locale="fr-FR">
+  <info>
+    <title>Vancouver DRSM (French)</title>
```

Comment:
Yep, DRSM is the name of my company. 
But there's 20 DRSM in France, all forming the Medical service of French national health insurance (French: "Service Médical de l'Assurance Maladie").
Since all DRSM are using this citation style, I think "Service Médical de l'Assurance Maladie (French)" would be better.
Can I change it to that with your instructions related to the ID and link above?